### PR TITLE
Update MK3_3.13.2 cmake from MK3 branch

### PR DESCRIPTION
**Pull request by 3d-gussner**
Cherry-pick failed so I copied needed changes.

### CODE REVIEW

File: Firmware/ultralcd.cpp [hunk 1]
```diff
@@ -1605,9 +1605,7 @@ static void lcd_support_menu()
 
   MENU_ITEM_BACK_P(PSTR("Firmware:"));
   MENU_ITEM_BACK_P(PSTR(" " FW_VERSION_FULL));
-#if (FW_DEV_VERSION != FW_VERSION_GOLD) && (FW_DEV_VERSION != FW_VERSION_RC)
-  MENU_ITEM_BACK_P(PSTR(" repo " FW_REPOSITORY));
-#endif
+//  MENU_ITEM_BACK_P(PSTR(" repo " FW_REPOSITORY));
```
Comment (vintagepc) [inline Firmware/ultralcd.cpp]: why was this disabled? CMake attempts to set it....

File: tests/CMakeLists.txt [hunk 2]
```diff
@@ -16,3 +16,27 @@ add_executable(tests ${TEST_SOURCES})
 target_include_directories(tests PRIVATE tests)
 target_link_libraries(tests Catch2::Catch2WithMain)
 catch_discover_tests(tests)
+
+set(ctest_test_args --output-on-failure)
+
+include(ProcessorCount)
```
Comment (vintagepc) [inline tests/CMakeLists.txt:22]: this is kind of unnecessary if using e.g. ninja over standard make

File: Firmware/Configuration.h [hunk 3]
```diff
@@ -5,61 +5,50 @@
...
+// Firmware version.
+// NOTE: These are ONLY used if you are not building via cmake and/or not in a git repository.
+// Otherwise the repository information takes precedence.
+#ifndef CMAKE_CONTROL
 #define FW_MAJOR 3
 #define FW_MINOR 13
-#define FW_REVISION 2
-#define FW_FLAVOR RC      //uncomment if DEBUG, DEVEL, ALPHA, BETA or RC
+#define FW_REVISION 0
+#warning "** Not sure why I had to touch this, but it seems like v3.13.1 is not in the linear history of this branch yet?"
```
Comment (vintagepc) [inline Firmware/Configuration.h]: this should probably be removed :rofl:

File: tests/CMakeLists.txt (hunk 2 above)
Comment (3d-gussner) [reply tests/CMakeLists.txt:22]: You added it https://github.com/prusa3d/Prusa-Firmware/commit/5ddac5cb6c725972bdf2f6861f4999ef13fe1193#diff-4461c617ceae0c7e0206622aacdf555aedd62eb129c2efb74c84fa1567bcbe0d

File: Firmware/ultralcd.cpp (hunk 1 above)
Comment (3d-gussner) [reply Firmware/ultralcd.cpp]: Fixed

File: Firmware/Configuration.h (hunk 3 above)
Comment (3d-gussner) [reply Firmware/Configuration.h]: Gone as it is annoying

File: tests/CMakeLists.txt (hunk 2 above)
Comment (DRracer) [reply tests/CMakeLists.txt:22]: I'd keep it, it does no harm and allows the good-old-make taking advantage of multiple cores

**Review by DRracer — APPROVED** [review]

**Pull request merged**